lib/diagnostic.js

```diff
@@ -18,7 +18,17 @@
 // Methods: sell, call
 
 // Answer Starts Here
-//
+const User = {
+  Name = 'Ryan Ongchoa',
+  email = 'ryan.ongchoa@libertymutual.com',
+  meals = [
+    {
+    breakfeast: "shake w/ banna"
+    lunch: "salad"
+    dinner: "steak"
```

Comment:
This is an object, so make sure that all your properties are separated by a comma.